src/components/tag.scss

```diff
@@ -0,0 +1,87 @@
+%tag-global {
```

Comment:
The implementation here should probably reflect the badge implementation. Aside from the hover style, this should be essentially the same.

Reply:
One different with this as opposed to badges is that these could potentially be used on buttons that are clickable and have a focus state so it may need to be a mixture of the button implementation and badge implementation. (obviously with the appropriate variables)